src/USAState/USAState.scss

```diff
@@ -0,0 +1,23 @@
+.title-info {
+  display: flex;
+  justify-content: center;
+}
+
+.state-image {
```

Comment:
Maybe we should look into changing this to an image that is being passed down rather than drawing out the state.  (At least in a future iteration)